website/docs/config/networks.mdx

```diff
@@ -85,15 +87,15 @@ Add the following entry ("testing": {...}) to the `network` property::
 {
    "network":{
       "ghostnet":{
-         "label":"Ghostnet Testnet",
+         "label":"Ghostnet Protocol Testnet",
          "rpcUrl":"https://rpc.ghostnet.teztnets.xyz",
          "protocol":"PtJakart2xVj7pYXJBXrqHgd82rdkLey5ZeeGwDgPp9rhQUbSqY",
```

Comment:
We should use the Alpha protocol here so that we don't have to keep updating this.